README.md

```diff
@@ -25,6 +25,7 @@ Thanks for using! :smile:
   - [Getting started (Native install)](#getting-started-native-install)
     - [Prerequisite](#prerequisite)
     - [Instructions](#instructions)
+      - [CentOS Installation](docs/guides/centos-installation.sh)
```

Comment:
You need to add a section into this document as it's a ToC. Please refer to the Docker section.